glib-macros/src/utils.rs

```diff
@@ -148,15 +148,63 @@ pub fn parse_item_attributes(attr_name: &str, attrs: &[Attribute]) -> Result<Vec
     Ok(v)
 }
 
-pub fn crate_ident_new() -> Ident {
+const KNOWN_GLIB_EXPORTS: [&str; 14] = [
+    // Current Re-exports from gtk-rs
+    "gtk",
+    "gio",
+    "gdk",
+    "gdk-pixbuf",
+    "gdkx11",
+    "graphene",
+    "pango",
+    "pangocairo",
+    // Current Re-exports from gtk4-rs
+    "gtk4",
+    "gsk4",
+    "gdk4",
+    "gdk4-wayland",
+    "gdk4-x11",
```

Comment:
Does it make sense to include these two (and `gdkx11` above)? You'll always have `gdk4` / `gdk` in your dependencies anyway.

Same for `pangocairo` and `pango`. They're more "extension" crates but you'd use the base one too always. Same thing with e.g. `libhandy` and `gtk` / `libadwaita` and `gtk4` and `gstreamer-*` and `gstreamer`.

Reply:
Yes, you can drop most of those and only keep gtk4/gdk4/gsk4 for gtk4-rs crates; gio/gdk for gtk-rs crates.

Reply:
Will do just to confirm new list: 
    "gtk",
    "gio",
    "gdk",
    "graphene",
    "pango",
    "gtk4",
    "gsk4",
    "gdk4",
    "gstreamer",

Reply:
looks fine to me, @sdroege ?

Reply:
Those plus `cairo` (note that it's called `cairo-rs`). Also `graphene` is called `graphene-rs`.

Reply:
I didn't put cairo in as glib was feature gated. If someone depended on cairo but not "use-glib" then it would try to use cairo and they wouldn't be able to fix it.

I guess they could directly depend on glib as a catchall

Reply:
The glib feature is enabled by default, so if they use `glib-macros` they probably didn't explictly disable the glib feature of cairo :)

Reply:
But maybe put cairo last because of this.